streampipes-client-python/streampipes_client/functions/machine_learning/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
lets move this into a dedicated module called `function_zoo`

Reply:
Should `function_zoo` be a submodule of the `functions` module. Or should it be seperated to differentiate between implementation and examples?

Reply:
I would see it as separate module on the same level like functions.
Should also be beneficial in terms of imports